src/test/java/jenkins/plugins/logstash/LogstashIntegrationTest.java

```diff
@@ -69,7 +69,7 @@ public void setup() throws Exception
     @Test
     public void buildWrapperOnMaster() throws Exception
```

Comment:
minor, but it would be nice to update the test name (not just s/wrapper/property/ buy maybe a different, more expressive name)